hack/validate-generated.sh

```diff
@@ -24,6 +24,10 @@ GENERATED_FILES=(
 	"pkg/addons/bindata.go"
 	"pkg/jsonpath/jsonpath.go"
 	"pkg/arm/bindata.go"
+	"pkg/util/mocks/mock_upgrade/types.go"
+	"pkg/util/mocks/mock_arm/arm.go"
+	"pkg/util/mocks/mock_azureclient/azureclient.go"
+	"pkg/util/mocks/mock_azureclient/mock_storage/storageclient.go"
```

Comment:
can we check the output of `git clean -ndx` or equivalent as a strategy here rather than maintaining this file list?

Reply:
i am now searching for the "generated by.."